### ISSUE TITLE
Milestone 3 Feedback

### ISSUE THREAD

**Issue by nafi007** [UBC-MDS/magmaviz#49]
Python package documentation: reasoning 

score drop:  Readthedocs.org documentation is not present or it is not possible to know so because the link it was not provided clearly in the readme or as a link - D

Everything else is ok. Good work!

**Comment by miyer26** [comment]
Hi, we just wanted to verify that you were able to view the correct [release](https://github.com/UBC-MDS/magmaviz/releases/tag/v0.3.7) corresponding to Milestone 3. The README file contained a button to Readthedocs.org in the header and also under the usage sections. Perhaps there was an additional expectation that we missed? Your clarification would be much appreciated. Thank you!

**Comment by nafi007** [comment]
You're right, I missed it. Your grade's been bumped up , apologies for the oversight!

**Comment by miyer26** [comment]
Thank you so much for verifying! Appreciate it!
